_shared/project/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:{{ python_versions()|first|pyformat(PyFormats.MAJOR_DOT_MINOR_DOT_PATCH_FMT) }}-alpine3.16
```

Comment:
There's nothing actually new about this `Dockerfile`, it's just the file that was previously at `pyramid-app/{{ cookiecutter.slug }}/Dockerfile` now moved into `_shared/` (and there's now a symlink at the file's original location, that's why git doesn't detect the file move).

Some files like this one that previously lived in the `pyramid-app` directory since they were only used by the `pyramid-app` cookiecutter are now moved into the `_shared` directory because they're now used by both `pyapp` and `pyramid-app`. This doesn't affect the `pypackage` cookiecutter: the `pypackage` directory doesn't contain symlinks to these shared files so it simply won't use them.